forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -206,14 +206,16 @@ fn apply_target(command: cmd::Deploy) -> Result<cmd::Deploy> {
         if command.target.is_some() {
             bail!("Both `--testnet` and `--target` were specified: must choose one")
```

Comment:
Do we also need to check if `--node-url` was supplied? It seems like you should only be able to use 1 of the three flags in a single command.

Reply:
Yeah that is correct. I did not realized this as this was already in place, fixing it now :)

Reply:
9494cf2 should add the checks for this